### ISSUE TITLE
Empty passphrase private key does not work with PoA.

### ISSUE THREAD

**Issue by MicahZoltu** [paritytech/parity#6799]
> I'm running:
>
> - **Parity version**: 1.8.0
> - **Operating system**: Linux
> - **And installed**: Docker

---

Attempting to use the empty-passphrase private key (`0x00a329c0648769a73afac7f9381e08fb43dbea72`) with Parity running with Aura PoA chain spec results in failure.  Setting everything up with that key as an unlocked account + single validator results in Parity erroring on startup that either the account being unlocked is not available (despite the key being in the key folder) or the the validator account not being available.

I recognize that the empty-passphrase key has caused trouble before and I suspect it is on an internal blacklist somewhere, but the error message should be _much_ more clear as to what the source of the problem is.  I spent about half a day troubleshooting my integration test setup to find out in the end that the problem was that I was using that private key instead of _any_ other private key.

**Comment by Office-Julia** [comment]
Hello.
Could you please share your configuration file?
Thanks. Julia.

**Comment by Office-Julia** [comment]
Also could you please provide chain specification to reproduce your issue?

**Comment by rphmeier** [comment]
Handling of the empty-recovery-phrase key should be clarified for sure. In general I'd recommend using any other key, just in case you forgot where the one you're using came from!

**Comment by MicahZoltu** [comment]
The below config and chain-spec files work fine as long as the private key referenced (`0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb` in this case) are anything _other_ than the empty passphrase private key.  If your private key is the empty passphrase private key and you replace `0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb` in the below files with `0x00a329c0648769a73afac7f9381e08fb43dbea72` (the public address of the empty phrase key) You end up getting incredibly uninformative error messages indicating that the key doesn't exist (not that it is blacklisted).  Its as though Parity simply "ignores" that private key when found in the keys directory and you are using anything other than Instant Seal config.

config.toml
```
[parity]
# Custom Chain
chain = "/parity/aura-chain-spec.json"
# Forces Parity to run even if there are known issues regarding consensus. Not recommended.
no_consensus = true
# Blockchain and settings will be stored in /parity.
base_path = "/parity"
# Parity databases will be stored in /parity/chains.
db_path = "/parity/chains"
# Your encrypted private keys will be stored in /parity/keys.
keys_path = "/parity/keys"

[account]
# From: [0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb] you'll be able to send tranasactions without password.
unlock = ["0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb"]
# File at /parity/password should contain passwords to unlock your accounts. One password per line.
password = ["/parity/password"]

[network]
# Enable or disable new peers discovery.
discovery = false
# Connect only to reserved nodes.
reserved_only = true

[rpc]
#  JSON-RPC will be listening for connections on IP 0.0.0.0.
interface = "all"
# Allows Cross-Origin Requests from domain '*'.
cors = "*"
# Only selected APIs will be exposed over this interface.
apis = ["all"]
# Allow connections only using specified addresses.
hosts = ["all"]

[websockets]
#  JSON-RPC will be listening for connections on IP 0.0.0.0.
interface = "all"
# Allows connecting from Origin '*'.
origins = ["all"]
# Only selected APIs will be exposed over this interface.
apis = ["all"]
# Allow connections only using specified addresses.
hosts = ["all"]

[ipc]
# You won't be able to use IPC to interact with Parity.
disable = true

[secretstore]
# You won't be able to encrypt and decrypt secrets.
disable = true

[ipfs]
# You won't be able to hash-query blockchain data.
disable = true

[dapps]
# You won't be able to access any web Dapps.
disable = true

[mining]
engine_signer = "0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb"
min_gas_price = 1
# Targeting 6500000 gas per block when sealing a new block.
gas_floor_target = "6500000"
# Gas limit will be raised at most by 6500000 gas.
gas_cap = "6500000"
# Parity will reject transactions above 6500000 gas.
tx_gas_limit = "6500000"
# Parity will not save local transaction queue to disk.
no_persistent_txqueue = true

[footprint]
# Significant speed up but unclean exit is unrecoverable (disables DB WAL).
fast_and_loose = true

[snapshots]
# Disables automatic periodic snapshots.
disable_periodic = true
```

chain-spec.json
```
{
    "name": "AuraChain",
    "engine": {
        "authorityRound": {
            "params": {
                "stepDuration": "1",
                "validators" : { "list": [ "0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb" ] }
            }
        }
    },
    "params": {
        "gasLimitBoundDivisor": "0x400",
        "accountStartNonce": "0x0",
        "maximumExtraDataSize": "0x20",
        "minGasLimit": "0x1",
        "networkID" : "0x2323",
        "eip150Transition": "0x0",
        "eip160Transition": "0x0",
        "eip161abcTransition": "0x0",
        "eip161dTransition": "0x0",
        "eip649Reward": "0x29A2241AF62C0000",
        "eip100bTransition": "0x0",
        "eip649Transition": "0x0",
        "eip98Transition": "0xffffffffffffffff",
        "eip140Transition": "0x0",
        "eip211Transition": "0x0",
        "eip214Transition": "0x0",
        "eip155Transition": "0x0",
        "eip658Transition": "0x0"
    },
    "genesis": {
        "seal": {
            "authorityRound": {
                "step": "0x0",
                "signature": "0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
            }
        },
        "difficulty": "0x20000",
        "gasLimit": "0x632EA0"
    },
    "accounts": {
        "0x0000000000000000000000000000000000000001": { "balance": "1", "builtin": { "name": "ecrecover", "pricing": { "linear": { "base": 3000, "word": 0 } } } },
        "0x0000000000000000000000000000000000000002": { "balance": "1", "builtin": { "name": "sha256", "pricing": { "linear": { "base": 60, "word": 12 } } } },
        "0x0000000000000000000000000000000000000003": { "balance": "1", "builtin": { "name": "ripemd160", "pricing": { "linear": { "base": 600, "word": 120 } } } },
        "0x0000000000000000000000000000000000000004": { "balance": "1", "builtin": { "name": "identity", "pricing": { "linear": { "base": 15, "word": 3 } } } },
        "0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb": { "balance": "0xffffffffffffffffffffffffffffffff"}
    }
}
```

**Comment by rphmeier** [comment]
> Its as though Parity simply "ignores" that private key when found in the keys directory 
That's exactly what happens. It was implemented intentionally that way, although it should be possible to relax it with an opt-in CLI flag.

**Comment by MicahZoltu** [comment]
Being able to relax it isn't my desire nor particularly important to me.  What is important I believe is some kind of messaging indicating that the key was ignored and why.  I spent hours debugging thinking it was something wrong with my config, my filesystem, directory permissions, etc. and I only discovered the source of the problem by happenstance.  If a message would have been logged on warn level at startup, I would have immediately known the problem and saved myself those hours.

Example message:
> Ignoring private key `<filename>/<public address>/<public key>` because it is the empty recovery seed key and has been the source of many lost funds.  See `<link to docs>` for details.
